tests/foreman/cli/test_environment.py

```diff
@@ -54,6 +54,108 @@ def test_positive_list_with_name(self):
                 self.assertEqual(len(result), 1)
                 self.assertEqual(result[0]['name'], name)
 
+    @tier2
+    @run_only_on('sat')
+    def test_positive_list_with_org_and_loc(self):
+        """Test Environment List filtering.
+
+        @id: 962c6750-f203-4478-8827-651db208ff92
+
+        @Assert: Results that match both organization and location are returned
+
+        @CaseLevel: Integration
+
+        @BZ: 1337947
+        """
+        # Create 2 envs with the same organization but different locations
+        org = make_org()
+        locs = [make_location() for _ in range(2)]
+        envs = [make_environment({
+            'name': gen_string('alpha'),
```

Comment:
nitpick: this line is redundant, factory will provide random name generated by fauxfactory anyways